### PR TITLE
[13] Style thE Product Details Page using CSS

### PR DESCRIPTION
**Pull request by opeyemipeter1759**
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by opeyemipeter1759** [comment]
@adaugochi kindly review my PR, thanks

File: submissions/opeyemiPeter/css/product.css [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+  align-items: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+.logo {
+  margin-right:auto;
+  margin-left: 75px;
+}
+.logo a {
+  font-family: "", cursive;
```
Comment (ToluAkin) [inline submissions/opeyemiPeter/css/product.css:35]: You already set the font family in the body tag. No need for it here. @opeyemipeter1759

File: submissions/opeyemiPeter/css/product.css [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+  color: #1b82d6;
+  font-weight: 100; 
+}
+.span-logo {
+  color: #1b82d6;
+  font-family: "Grenze Gotisch", cursive;
```
Comment (ToluAkin) [inline submissions/opeyemiPeter/css/product.css:41]: You already set the font family in the body tag. No need for it here. @opeyemipeter1759

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Well done @opeyemipeter1759

File: submissions/opeyemiPeter/css/product.css (hunk 2 above)
Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/css/product.css:41]: @ToluAkin the font-family is for span in the logo, it was intended to be different from that of the rest, reasons why i declared it there.

File: submissions/opeyemiPeter/css/product.css (hunk 1 above)
Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/css/product.css:35]: @ToluAkin i wanted a different fontfamily for the logo from that of the family

File: submissions/opeyemiPeter/css/product.css (hunk 2 above)
Comment (opeyemipeter1759) [reply submissions/opeyemiPeter/css/product.css:41]: @ToluAkin good morning, sequel to your comment on my PR, the font family i wanted for the logo was suppose to be entirely different from that of the rest, reasons why i declared a different font family for the logo.

**Review by ToluAkin — APPROVED** [review]

**Comment by opeyemipeter1759** [comment]
thanks.

**Pull request merged**